Resfeber/View Controllers/TripsViewController.swift

```diff
@@ -10,9 +10,11 @@ import UIKit
 
 class TripsViewController: UIViewController {
     // MARK: - Properties
-    
+
     fileprivate let destinationController = DestinationController()
     fileprivate var collectionView: UICollectionView!
+    fileprivate var coreDataStack = CoreDataStack()
```

Comment:
Singleton! Setting up multiple instances of the CoreDataStack is going to produce some really bad behaviors